### PR TITLE
[sflow + dropmon] added patches for the sFlow + dropmon feature.

### PR DESCRIPTION
**Pull request by vadymhlushko-mlnx**
#### Why I did it
`sFlow + dropmon` feature require the patches provided by SDK team [Mellanox/Switch-SDK-drivers/tree/sx_sdk_4_5_1200_4.5.1208/kernel_backports/5.10.0](https://github.com/Mellanox/Switch-SDK-drivers/tree/sx_sdk_4_5_1200_4.5.1208/kernel_backports/5.10.0)

#### How I did it
Added patches.

### CODE REVIEW

**Comment by saiarcot895** [comment]
A few comments:

1. Split the patch you have into two (or more) different patches: one that is backported from upstream (with the commit ID included in the patch), and one that includes any custom changes.
2. In each patch file, describe what changes are being done, with a useful subject line.
3. Is there a reason that the patch has to be conditionally included here, and why it can't be included always?

**Comment by vadymhlushko-mlnx** [comment]
> A few comments:
> 
> 1. Split the patch you have into two (or more) different patches: one that is backported from upstream (with the commit ID included in the patch), and one that includes any custom changes.
> 2. In each patch file, describe what changes are being done, with a useful subject line.
> 3. Is there a reason that the patch has to be conditionally included here, and why it can't be included always?

Fixed, could you please review

**Comment by liat-grozovik** [comment]
@saiarcot895 kindly reminder to review the changes following comments provided.

File: patch/0003-psample-added-define-PSAMPLE_MD_EXTENDED_ATTR.patch [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+diff --git a/include/net/psample.h b/include/net/psample.h
+index e328c5127..b7c79f634 100644
+--- a/include/net/psample.h
++++ b/include/net/psample.h
+@@ -14,6 +14,8 @@ struct psample_group {
+ 	struct rcu_head rcu;
+ };
+ 
++#define PSAMPLE_MD_EXTENDED_ATTR   1
```
Comment (saiarcot895) [inline patch/0003-psample-added-define-PSAMPLE_MD_EXTENDED_ATTR.patch:19]: Where is this used?

Comment (vadymhlushko-mlnx) [reply patch/0003-psample-added-define-PSAMPLE_MD_EXTENDED_ATTR.patch]: Seems like it was never used - deleted patch

**Comment by vadymhlushko-mlnx** [comment]
> Please add the upstream commit ID in the description for `0001-psample-Encapsulate-packet-metadata-in-a-struct.patch` and `0002-psample-Add-additional-metadata-attributes.patch`.

Fixed

**Pull request merged**